src/test/v2.0/advanced/fdc3.open.ts

```diff
@@ -20,15 +22,20 @@ export default () =>
     getCommonOpenTests(control, openDocs, config);
 
     //run v2.0-only open tests
-    const AOpensBWithWrongContext = "(2.0-AOpensBWithWrongContext) Received App timeout when opening app B with fake context, app b listening for different context";
-    it(AOpensBWithWrongContext, async () => {
-      await control.addListenerAndFailIfReceived();
-      await expectAppTimeoutErrorOnOpen(openApp.d.id);
-      await control.closeMockApp(AOpensBWithWrongContext);
-    }).timeout(constants.NoListenerTimeout + 1000);
 
-    const AOpensBNoListen = "(2.0-AOpensBNoListen) Received App timeout when opening app B with fake context, app b not listening for any context";
-    it(AOpensBNoListen, async () => {
-      await expectAppTimeoutErrorOnOpen(openApp.e.id);
-    }).timeout(constants.NoListenerTimeout + 1000);
+    const AOpensB4 = "(AOpensB4) Can open app B from app A with appId as config.target, and recieves the same appId and also contains InstanceId";
+    it(AOpensB4, async () => {
+      const result = control.contextReceiver("fdc3-conformance-opened");
+      const targetApp = control.createTargetApp(openApp.b.name,openApp.b.id);
```

Comment:
you shouldn't use this here - createTargetApp should only be used for AOpensXXX3

Reply:
AOpensB3
AFailsToOpenB3:
AOpensBWithContext3
AOpensBWithWrongContext